db/migrate/20230629073203_create_identities.rb

```diff
@@ -0,0 +1,12 @@
+class CreateIdentities < ActiveRecord::Migration[7.0]
+  def change
+    create_table :identities do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :email, null: false
+      t.string :password_digest, null: false
+      t.string :provider, null: false
```

Comment:
We should have the following:

```
provider_uid -> nullable but to store third-party IDs
identifier -> non-nullable  as it can be email or any other value used by the identify provider 
credential -> nullable as it will be empty for email+password auth (for Facebook we can use that token to fetch user data) 
```

Reply:
@olivierobert I tried removing `password_digest`, but it results to an error unless I add it 💭

Reply:
I think you need to modify this code as well
https://github.com/nimblehq/ic-rails-huey-toby/pull/44/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR30

Reply:
@hoangmirs Yes, correct. I removed it there as well 🙏 But still the same error

![Screen Shot 2023-07-06 at 17 20 22](https://github.com/nimblehq/ic-rails-huey-toby/assets/18277915/9ed803eb-9081-434a-a33b-af29f8328876)